frontend/src/e2e-playwright/pages/employee/child-information.ts

```diff
@@ -0,0 +1,90 @@
+import { Page } from 'playwright'
+import { RawElement } from 'e2e-playwright/utils/element'
+
+export type Collapsible = 'dailyServiceTimes'
+
+export default class ChildInformationPage {
+  constructor(private readonly page: Page) {}
+
+  readonly #collapsibles: Record<Collapsible, RawElement> = {
+    dailyServiceTimes: new RawElement(
+      this.page,
+      '[data-qa="child-daily-service-times-collapsible"]'
+    )
+  }
+
+  async openCollapsible(collapsible: Collapsible) {
+    const element = this.#collapsibles[collapsible]
+    await element.click()
+    return element
+  }
+
+  async openDailyServiceTimesCollapsible() {
```

Comment:
Aren't `openCollapsible` and `openDailyServiceTimesCollapsible` doing the same thing?

Reply:
Tweaked a bit, hope it's better now!